### DISCUSSION TITLE
只要接受了这个设定。。。

### DISCUSSION THREAD

**Discussion by MRScongZZZ** [share121/inter-knot#2205] [其他]
![47e48cd0ada60fbd78afbd84f184788](https://github.com/user-attachments/assets/86ae8294-e42f-43f2-9113-d4c1ece5a5a4)
![dbf0d2579ced478b9b95e995a8b0e45](https://github.com/user-attachments/assets/ad9308ff-bd14-4f9f-9a73-61c81ee13bb5)

**Comment by poolofwater** [comment]
老太婆可爱捏
<img src="https://img-static.mihoyo.com/communityweb/upload/81ed7cd6675af50aec9cc622bda99b88.png" alt="比利_害羞" width="75" >

**Comment by NOVAQUQ** [comment]
![-3d875b4c01d9619d](https://github.com/user-attachments/assets/b2484c76-d87a-43d8-b89c-193e6faea33c)
那……那我就是……无敌的？
